### PR TITLE
miscellanous stuff

### PR DESCRIPTION
**Pull request by austin987**
Misc patches from various branches that are simple/straightforward enough to (IMO) upstream without separate PRs/issues.

If you disagree, let me know.

### CODE REVIEW

File: scripts/InstallScripts/InstallPrawnOS.sh [hunk 1]
```diff
@@ -221,7 +230,6 @@ external_partition() {
     EXTERNAL_TARGET=$1
     kernel_start=8192
     kernel_size=65536
-    root_start=$(($kernel_start + $kernel_size))
```
Comment (SolidHal) [inline scripts/InstallScripts/InstallPrawnOS.sh:224]: I'll have to review this closer. 
Might be related to usb install not functioning.

**Comment by SolidHal** [comment]
This looks great overall, thanks for collecting these changes. Just have to look over that InstallPrawnOS.sh change.

**Comment by austin987** [comment]
If you'd prefer, I can drop that one for now and resubmit it later with other shellcheck fixes.

**Comment by SolidHal** [comment]
since you closed #169, this looks good

**Pull request merged**